test.js

```diff
@@ -581,7 +581,12 @@ var BASIC_TESTS = [
   ['camelCase', 'camelCases'],
   ['PascalCase', 'PascalCases'],
   ['Alumnus', 'Alumni'],
-  ['CHICKEN', 'CHICKENS']
+  ['CHICKEN', 'CHICKENS'],
+  ['日本語', '日本語'],
+  ['한국', '한국'],
+  ['中文', '中文'],
+  ['اللغة العربية', 'اللغة العربية'],
```

Comment:
Right to left inside a left to right document is really confusing, and the GitHub UI does not make it easier here 👍